XamlControlsGallery/ControlPages/CornerRadiusPage.xaml

```diff
@@ -0,0 +1,73 @@
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:local="using:AppUIBasics"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}"
+    mc:Ignorable="d">
+    <StackPanel>
+        <TextBlock IsTextSelectionEnabled="True">
+            <LineBreak />
+            <Run Text="For basic UI elements like buttons that users see on screen at all times," />
+            <Run Text="the default radius value is" />
+            <Run FontWeight="bold" Text="2px" />
+            <Run Text="." />
+            <LineBreak />
+            <LineBreak />
+            <Run Text="For transient UI elements that appear on screen temporarily, like MenuFlyout, or elements that overlay other UI, like TabView tabs," />
+            <LineBreak />
+            <Run Text="the default radius value we use for is" />
+            <Run FontWeight="Bold" Text="4px" />
+            <Run Text="." />
+        </TextBlock>
+        <local:ControlExample
+            x:Name="Example1"
+            HeaderText="A button with CornerRadius"
+            MinimumUniversalAPIContract="7">
+            <Button contract7Present:CornerRadius="5" Content="MyButton" />
+            <local:ControlExample.Xaml>
+                <x:String>
+                    &lt;Button conditionalNameSpace:CornerRadius=&quot;5&quot; Content=&quot;MyButton&quot;/&gt;
+                </x:String>
+            </local:ControlExample.Xaml>
+        </local:ControlExample>
+        <local:ControlExample
+            x:Name="Example2"
+            HeaderText="A button without CornerRadius"
+            MinimumUniversalAPIContract="7">
+            <Button contract7Present:CornerRadius="0" Content="MyButton" />
+            <local:ControlExample.Xaml>
+                <x:String>
+                    &lt;Button conditionalNameSpace:CornerRadius=&quot;0&quot; Content=&quot;MyButton&quot;/&gt;
+                </x:String>
+            </local:ControlExample.Xaml>
+        </local:ControlExample>
+        <local:ControlExample MinimumUniversalAPIContract="7">
+            <Button contract7Present:CornerRadius="5,0,5,0" Content="CornerRadius: 5,0,5,0" />
+        </local:ControlExample>
+        <local:ControlExample HeaderText="A textbox with CornerRadius" MinimumUniversalAPIContract="7">
+            <TextBox contract7Present:CornerRadius="10" Header="A textbox with CornerRadius" />
+                <local:ControlExample.Xaml>
+                <x:String>
+                    &lt;TextBox conditionalNameSpace:CornerRadius&quot;10&quot; Header=&quot;A textbox with CornerRadius&quot; /&gt;
+                </x:String>
+            </local:ControlExample.Xaml>
+        </local:ControlExample>
+
+        <TextBlock Text="CornerRadius values for Overlay controls, such as content dialogs, flyouts and other popups can be set in App.xaml with property OverlayCornerRadius ." />
```

Comment:
I think we should show a sample for this though, where you can open a flyout or something like this, or at least show the OverlayCornerRadius resource i.e. how to change it's value in the sample code are.

Reply:
I have this in App.xaml:
```xaml
<ResourceDictionary>
                    <CornerRadius x:Key="OverlayCornerRadius ">10</CornerRadius>
 </ResourceDictionary>
```
but it dosen't work @chingucoding

Reply:
I think there is a space to much in the name, it should be the following:

```xaml
<ResourceDictionary>
             <CornerRadius x:Key="OverlayCornerRadius ">10</CornerRadius>
 </ResourceDictionary>
```

One thing I noticed is that MenuFlyouts don't seem to respect that. @ranjeshj Do you know if this is intentional?

Reply:
Now I have:
```xaml
<CornerRadius x:Key="OverlayCornerRadius">10</CornerRadius>
```
but it still does not work. Is there anything I need to set _within_ the dialog?

Reply:
It turns out that for the ContentDialog, it seems that you need to set it explicitly.

Reply:
How? Could you give me an example? Also, if I set OverlayCornerRadius, all the overlays will become rounded.

Reply:
Yes if you change the OverlayCornerRadius in App.xaml, this will change the corner radius of all controls using that, e.g. MenuFlyout or TabView.

Regarding ContentDialog, thiscode gives you the following ContentDialog:
```c#
var contentDialog = new ContentDialog()
{
    Content = "I am a dialog",
    CornerRadius = new CornerRadius(20),
    CloseButtonText = "Awesome!"
};

contentDialog.ShowAsync();

```

![image](https://user-images.githubusercontent.com/16122379/89928758-98d27580-dc08-11ea-8f0f-1412245054e1.png)